06-raster-vector.Rmd

```diff
@@ -311,12 +311,14 @@ rast_poly_nlcd = tm_shape(nlcd) +
 tmap_arrange(rast_poly_srtm, rast_poly_nlcd, ncol = 2)
 ```
 
+Although the `terra` package offers rapid extraction of raster values within polygons, the `extract()` can still be a bottleneck when processing large polygon datasets. The `exactextractr` package offers a [significantly faster alternative](https://github.com/Robinlovelace/geocompr/issues/813) for extracting pixel values through the `exact_extract()` function. The `exact_extract()` function also computes by default the fraction of each raster cell overlapped by the polygon, which is more precise (see note below for details). 
```

Comment:
Looks good but please switch to sentence-based line breaks. See our style guide for further info.